Community guilds

advaith1 opened the pull request:
https://github.com/discord/discord-api-docs/issues/1757#issuecomment-652079149

advaith1 committed:
document COMMUNITY guild feature

MinnDevelopment commented on docs/resources/Guild.md:
```diff
@@ -115,6 +115,7 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 | VERIFIED               | guild is verified                                                               |
 | PARTNERED              | guild is partnered                                                              |
 | PUBLIC                 | guild is public                                                                 |
+| COMMUNITY              | guild is a community                                                            |
```
This doesn't really explain anything

advaith1 replied:
The other ones don't really explain anything either, like "guild is public", so I assumed that's how Discord wants it

MinnDevelopment replied:
At this point you might as well remove the description column and just list the feature names. This shouldn't become the standard.

advaith1 committed:
put a useful description for community

not gonna touch public and public_disabled for now, they'll probably be removed soon

advaith1 committed:
clarify wording

advaith1 committed:
lurking is now tied to discoverable

advaith1 committed:
public -> community & discoverable

ThaTiemsz commented on docs/resources/Guild.md:
```diff
@@ -107,23 +107,23 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
+| VERIFIED               | guild is verified                                                               |
+| PARTNERED              | guild is partnered                                                              |
+| PUBLIC                 | guild is public                                                                 |
+| COMMUNITY              | guild can enable welcome screen and discovery, and receives community updates   |
```
```suggestion
| COMMUNITY              | guild can enable welcome screen and discovery, and receive community updates   |
```

advaith1 replied:
I was thinking of that as in "the guild receives community updates" since it's required for community guilds

advaith1 committed:
Merge branch 'master' into community

advaith1 committed:
public -> community in examples

advaith1 committed:
Merge branch 'master' into community

advaith1 committed:
Merge branch 'master' into community
